package.json

```diff
@@ -40,33 +40,33 @@
   },
   "extensionDependencies": [],
   "dependencies": {
-    "diff": "^4.0.1",
+    "diff": "^4.0.2",
     "json-rpc2": "^1.0.2",
     "moment": "^2.24.0",
-    "semver": "^6.3.0",
+    "semver": "^7.3.2",
     "tree-kill": "^1.2.2",
-    "vscode-debugadapter": "^1.36.0",
-    "vscode-debugprotocol": "^1.36.0",
+    "vscode-debugadapter": "^1.40.0",
+    "vscode-debugprotocol": "^1.40.0",
     "vscode-extension-telemetry": "^0.1.2",
     "vscode-languageclient": "6.1.0",
     "web-request": "^1.0.7"
   },
   "devDependencies": {
-    "@types/fs-extra": "^8.0.0",
+    "@types/fs-extra": "^8.1.0",
     "@types/glob": "^7.1.1",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^12.7.2",
-    "@types/semver": "^6.0.1",
-    "@types/sinon": "^7.5.2",
-    "@types/vscode": "^1.25.0",
-    "fs-extra": "^8.1.0",
-    "glob": "^7.1.4",
-    "mocha": "^6.2.0",
-    "prettier": "^1.16.4",
-    "sinon": "^9.0.1",
-    "tslint": "^5.19.0",
-    "typescript": "^3.7.2",
-    "vscode-test": "^1.2.3"
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^13.11.1",
+    "@types/semver": "^7.1.0",
+    "@types/sinon": "^9.0.0",
+    "@types/vscode": "^1.44.0",
```

Comment:
Turns out min version specified for `@types/vscode` cannot be greater than what we have in engines, the vsce command begins to fail. Also, we missed getting the lock file updated in this PR. Fixed both in https://github.com/microsoft/vscode-go/commit/1c314d4b6094d0be7cd07ce718657d50f4636423